src/components/registration-lite.js

```diff
@@ -369,7 +375,12 @@ const RegistrationLite = (
                                         closeWidget={closeWidget}
                                         hasVirtualAccessLevel={hasVirtualAccessLevel}
                                         supportEmail={supportEmail}
-                                        footerHasTicketText={rest.footerHasTicketText}
+                                        initialOrderComplete1stParagraph={initialOrderComplete1stParagraph}
```

Comment:
this is not correct
please check strategy used here
https://github.com/fntechgit/summit-registration-lite/blob/2d8a39f6f09729c69131c526732ccc6bff598fcc/src/components/registration-lite.js#L372